substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/container/ContainerLibrary.java

```diff
@@ -48,46 +48,46 @@
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/os/posix/os_posix.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/os/posix/os_posix.inline.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/memory/allocation.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/memory/allocation.inline.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/memory/allStatic.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/nmt/memflags.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/runtime/os.cpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/runtime/os.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/runtime/os.inline.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/checkedCast.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/compilerWarnings_gcc.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/compilerWarnings.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/globalDefinitions_gcc.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/globalDefinitions.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/macros.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/ostream.cpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/ostream.hpp")
 // The following annotations are for files in `src/svm`, which are completely customized for SVM
-@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+5/src/hotspot/share/logging/log.hpp")
-@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+9/src/hotspot/share/memory/allocation.cpp")
-@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+12/src/hotspot/share/runtime/globals.hpp")
-@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+5/src/hotspot/share/utilities/debug.cpp")
-@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+5/src/hotspot/share/utilities/debug.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/logging/log.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/memory/allocation.cpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/runtime/globals.hpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/debug.cpp")
+@BasedOnJDKFile("https://github.com/openjdk/jdk/blob/jdk-24+15/src/hotspot/share/utilities/debug.hpp")
```

Comment:
@jerboaa fyi, we have update libsvm_container to include "[JDK-8322420](https://bugs.openjdk.org/browse/JDK-8322420): [Linux] cgroup v2: Limits in parent nested control groups are not detected" <https://github.com/openjdk/jdk/pull/20646>

Reply:
OK, thanks!